baselines/dom.generated.d.ts

```diff
@@ -11377,7 +11389,7 @@ declare var ReadableStream: {
 };
 
 interface ReadableStreamBYOBReader extends ReadableStreamGenericReader {
-    read(view: ArrayBufferView): Promise<ReadableStreamReadResult<ArrayBufferView>>;
+    read<T extends ArrayBufferView>(view: T): Promise<ReadableStreamReadResult<T>>;
```

Comment:
When you pass a `Uint8Array`, you're guaranteed to receive a `Uint8Array` back in the result. This type reflects that. 🙂

Reply:
Also a good candidate for a unit test.